docs/reference/commands.md

```
@@ -124,32 +124,71 @@ sudo Mac-Clean --dry-run --json
+# Sum of all estimated bytes for would_run categories
+sudo Mac-Clean --dry-run --json | jq '[.results.categories.details | to_entries[] | select(.value.status == "would_run" and .value.estimated_bytes != null) | .value.estimated_bytes] | add'
+```
+
+Note: `estimated_bytes` / `estimated_human` are populated only for
+categories whose section body calls `record_category_size`. As of
+v5.7.0, that's categories #29 (Browser Testing Tool Caches), #30
+(Crash Reports), and #31 (User Tool Caches). Other categories still
+appear in `details` with just `status` and `skip_flag`.
+  }
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)